### DISCUSSION TITLE
PNG Images processing error

### DISCUSSION THREAD

**Discussion by balamundru** [ImageMagick/ImageMagick#6390] [Help]
### ImageMagick version

7.1.1-4

### Operating system, version and so on

redhat 8.6

### Description

convert: symbol lookup error: /usr/local/lib/libMagickCore-7.Q16HDRI.so.10: undefined symbol: png_get_eXIf_1, version PNG16_0

I am getting above error while trying to resize png images.

Imagemagick version 7.1.1-4
libpng version: 1.6.31

**Comment by snibgo** [comment]
What is your command? What is your input image?

Do you get the error with all PNG files or just some?

**Comment by dlemstra** [comment]
You will either need to upgrade libpng or ImageMagick. We added a check for the version after this release.
